src/back/importGame.ts

```diff
@@ -223,7 +223,7 @@ async function createGameFromCurationMeta(gameId: string, gameMeta: EditCuration
     dateAdded:           date.toISOString(),
     dateModified:        date.toISOString(),
     broken:              false,
-    extreme:             !!strToBool(gameMeta.extreme || ''),
+    extreme:             gameMeta.extreme || false,
```

Comment:
I don't think it's a good idea to go from parsing ``yes`` and ``true`` (both case insensitive) as true to only parsing``Yes`` (case sensitive) as true. This could break curations written or edited by hand.